3ds Max/Max2Babylon/Forms/ExporterForm.cs

```diff
@@ -79,6 +79,8 @@ private void ExporterForm_Load(object sender, EventArgs e)
             Tools.PrepareCheckBox(chkKHRTextureTransform, Loader.Core.RootNode, "babylonjs_khrTextureTransform");
             Tools.PrepareCheckBox(chkKHRMaterialsUnlit, Loader.Core.RootNode, "babylonjs_khr_materials_unlit");
             Tools.PrepareCheckBox(chkExportMaterials, Loader.Core.RootNode, "babylonjs_export_materials", 1);
+            Tools.PrepareCheckBox(chkExportMorphTangents, Loader.Core.RootNode, "babylonjs_export_Morph_Tangents", 0);
```

Comment:
Currently we also have a checkbox for exporting morph target tangents and normals in the other exporter form @PatrickRyanMS , would it be alright to move them to the main exporter dialog?